ci-operator/jobs/infra-periodics.yaml

```diff
@@ -271,6 +271,43 @@ periodics:
         - secret:
             name: cluster-secrets-aws
     serviceAccountName: ipi-deprovisioner
+- agent: kubernetes
+  labels:
+    ci.openshift.io/role: infra
+  cron: ""
+  decorate: true
+  interval: 12h
+  name: periodic-prow-image-autobump
+  spec:
+    containers:
+    - command:
+      - /usr/bin/autobump.sh
+      image: registry.svc.ci.openshift.org/ci/autobumper:latest
+      imagePullPolicy: Always
+      name: "autobump"
+      resources:
+          requests:
+            cpu: 500m
+      env:
+      ### TODO: update those tbu(s)
+      - name: github_login
```

Comment:
We can hard-code this to be `openshift-merge-robot` (and in general should not need it)

Reply:
Removed.